### PR TITLE
trans_hugepage_memory_stress:fix Invalid argument

### PR DESCRIPTION
**Pull request by zhenyzha**
trans_hugepage_memory_stress:fix Invalid argument

ID:2061631
Signed-off-by: zhenyzha <zhenyzha@redhat.com>

### CODE REVIEW

**Comment by zhenyzha** [comment]
`(1/1) Host_RHEL.m9.u0.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.9.0.0.aarch64.io-github-autotest-qemu.trans_hugepage.memory_stress.arm64-pci: PASS (345.98 s)`

@PaulYuuu Could you pls help review when you are free. thanks.

File: generic/tests/trans_hugepage_memory_stress.py [hunk 1]
```diff
@@ -42,7 +43,7 @@ def run(test, params, env):
         mem_swap = int(utils_memory.read_from_meminfo('SwapFree')) / 1024
         hugepage_size = (int(utils_memory.read_from_meminfo('Hugepagesize')) /
                          1024)
-        nr_hugetlbfs = (mem_free + mem_swap - mem - qemu_mem) / hugepage_size
+        nr_hugetlbfs = math.ceil((mem_free + mem_swap - mem - qemu_mem) / hugepage_size)
```
Comment (PaulYuuu) [inline generic/tests/trans_hugepage_memory_stress.py:46]:
```suggestion
        nr_hugetlbfs = (mem_free + mem_swap - mem - qemu_mem) // hugepage_size
```

Comment (zhenyzha) [reply generic/tests/trans_hugepage_memory_stress.py:46]: Use` // `to take the remainder, but the page needs to be larger than the remainder. So use `ceil` here.

Comment (PaulYuuu) [reply generic/tests/trans_hugepage_memory_stress.py:46]: `math.ceil()` method rounds a number UP to the nearest integer, but the behavior of `//` rounds a number DOWN, which aligns to `/` in python2.
So this is a change in history, any background? @huangyum , what's your opinion?

Comment (huangyum) [reply generic/tests/trans_hugepage_memory_stress.py:46]: We should set the hugepage number at least as required. So I prefer to use math.ceil. Thanks.

Comment (PaulYuuu) [reply generic/tests/trans_hugepage_memory_stress.py:46]: OK, thanks all.

**Review by PaulYuuu — APPROVED** [review]

**Pull request merged**